### PR TITLE
add new relese v0.6.0

### PR DESCRIPTION
**Pull request by samzong**
Signed-off-by: samzong.lu <samzong.lu@gmail.com>

### CODE REVIEW

File: docs/zh/docs/baize/intro/release-notes.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 本页列出智能算力各版本的 Release Notes，便于您了解各版本的演进路径和特性变化。
 
+## 2024-06-30
+
+### v0.6.0
+
+#### 功能
+
+- **新增** 支持 创建 `Code Server` 类型的 `Notebook`，提供原生 `VS Code` 开发体验。
```
Comment (kebe7jun) [inline docs/zh/docs/baize/intro/release-notes.md:11]:
```suggestion
- **新增** 支持 创建 `Code` 类型的 `Notebook`，提供原生 `VS Code` 开发体验。
```

界面是 Code

File: docs/en/docs/baize/intro/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,24 @@ date: 2024-05-21
 This page lists the Release Notes for Intelligent Engine,
 so that you can learn its evolution path and feature changes.
 
+## 2024-06-30
+
+### v0.6.0
+
+#### Features
+
+- **New**: Support for creating `Code Server` type `Notebook`, providing a native `VS Code` development experience.
```
Comment (kebe7jun) [inline docs/en/docs/baize/intro/release-notes.md:17]:
```suggestion
- **New**: Support for creating `Code` type `Notebook`, providing a native `VS Code` development experience.
```

File: docs/en/docs/baize/intro/release-notes.md [hunk 3]
```diff
@@ -22,22 +40,22 @@ so that you can learn its evolution path and feature changes.
...
-- **Added** **`Notebook` now supports local SSH access, compatible with various development tools such as `Pycharm`, `VS Code`, etc.**
-- **Added** **Upgrade `Notebook` image to support the built-in `CLI` tool `baizectl`, for command-line task submission and management.**
+- **Added** `Notebook` now supports local SSH access, compatible with various development tools such as `Pycharm`, `VS Code`, etc.
+- **Added** Upgrade `Notebook` image to support the built-in `CLI` tool `baizectl`, for command-line task submission and management.
 - **Added** `Notebook` adds affinity scheduling strategy configuration.
 - **Added** Distributed training tasks can now configure `SHM size` through the UI.
 - **Added** One-click restart function for training tasks.
-- **Added** **Model training tasks support custom cluster scheduler specification.**
-- **Added** **Training task analysis tool `Tensorboard` support, can be launched with one click in `Notebook` and training tasks.**
+- **Added** Model training tasks support custom cluster scheduler specification.**
```
Comment (windsonsea) [inline docs/en/docs/baize/intro/release-notes.md:57]:
```suggestion
- **Added** Model training tasks support custom cluster scheduler specification.
```

File: docs/en/docs/baize/intro/release-notes.md [hunk 4]
```diff
@@ -8,6 +8,24 @@ date: 2024-05-21
...
+### v0.6.0
+
+#### Features
+
+- **New**: Support for creating `Code Server` type `Notebook`, providing a native `VS Code` development experience.
+- **New**: Support for quickly copying `Notebook`.
+- **New**: When selecting a work cluster, display the cluster's status information, making it unselectable if it is disconnected or offline.
+- **New**: When creating inference services, support using `vLLM` as the inference engine, exposing native `vLLM` capabilities.
+- **New**: When creating inference services, `vLLM` supports configuring `Lora` inference parameters.
+- **Optimization**: When creating a `Notebook`, the default queue priority is adjusted to `High`.
```
Comment (windsonsea) [inline docs/en/docs/baize/intro/release-notes.md:22]:
```suggestion
- **New**: Support for quickly copying `Notebook`.
- **New**: When selecting a work cluster, display the cluster's status information, making it unselectable if it is disconnected or offline.
- **Added** support using `vLLM` as the inference engine, exposing native `vLLM` capabilities when creating inference services.
- **Added** `vLLM` supports configuring `Lora` inference parameters when creating inference services.
- **Optimized** the default queue priority `High` when creating a `Notebook`.
```
v0.6.0 风格跟以前不太一样了

**Comment by samzong** [comment]
All fixed. @kebe7jun @windsonsea

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**